### PR TITLE
Test ambiguate

### PR DESCRIPTION
**Pull request by mmozerwayfair**
1. Add tests for _disambugate.py
2. update docstrings

The last function in disambiguate.py is deprecated and has been removed in a separate pull-request, that is not yet merged.
Keeping it here to avoid merge conflicts later. Just don't wonder that the test for that function is missing

### CODE REVIEW

File: tests/test_disambiguate.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+
+def test__best_cluster():
+    # a ~realistic clusterting example
+    vectors = []
+    for i in range(100):
+        vectors.append(
+            [math.sin(math.pi * 2 * i / 100), math.cos(math.pi * 2 * i / 100)]
+        )
+    print(vectors)
```
Comment (romatik) [inline tests/test_disambiguate.py:67]: don't need print here

File: tests/test_disambiguate.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        vec,
+    )
+    assert (
+        aspects[0] == "chair"
+        and aspects[1] == "sofa"
+        and aspects[2] == "table"
+        and aspects[3] == "asdf"
+        and synsets[0].name() == "chair.n.01"
+        and synsets[1].name() == "sofa.n.01"
+        and synsets[2].name() == "table.n.03"  # disambiguation failure
```
Comment (romatik) [inline tests/test_disambiguate.py:114]: why is this a failure?

Comment (mmozerwayfair) [reply tests/test_disambiguate.py:114]: it's fairly minor.
With the other terms, I was hoping for
"table.n.02" 'a piece of furniture having a smooth flat top that is usually supported by one or more vertical legs'
"table.n.03" is 'a piece of furniture with tableware for a meal laid out on it'

Comment (romatik) [reply tests/test_disambiguate.py:112]: eh, close enough :)

**Review by romatik — APPROVED** [review]

**Pull request merged**